bedrock/mozorg/templates/mozorg/about/index-m24.html

```diff
@@ -16,7 +16,8 @@
 {% block body_id %}about{% endblock %}
 
 {% block page_css %}
-  {{ css_bundle('m24') }}
+  {{ css_bundle('m24-root') }}
```

Comment:
Shower thought: if we do want to avoid the repetition, we could maybe do it with some switch logic like
```suggestion
{% if not switch('m24-navigation-and-footer') %}
  {{ css_bundle('m24-root') }}
{% endif %}
```

Reply:
Oh, there is an issue to remove the nav+footer style redundancy https://github.com/mozilla/bedrock/issues/15120